src/bundle/Resources/views/content/tab/versions/table.html.twig

```diff
@@ -30,11 +30,11 @@
             path('ez_user_update', {
             'contentId': version.contentInfo.id,
             'versionNo': version.versionNo,
-            'language': version.translations[0].languageCode,
+            'language': version.initialLanguageCode,
         }) : path('ez_content_draft_edit', {
             'contentId': version.contentInfo.id,
             'versionNo': version.versionNo,
-            'language': version.translations[0].languageCode,
+            'language': version.initialLanguageCode,
```

Comment:
We need a fallback here as well since `initialLanguageCode` can be `null` in rare cases.

Reply:
Did not know that, thanks! Fixed in the newest commit.